src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -219,6 +220,7 @@ private void SetIds(DatabaseTopology topology, string shardedDatabaseId)
 
         public SubscriptionStorage SubscriptionStorage { get; }
 
+        public string Parent { get; }
```

Comment:
parent? what does it mean? rename and add a description here

Reply:
this is the db name without the shard index for `Northwind$0` =>  `Northwind`

I renamed this property